src/styles/variables.scss

```diff
@@ -0,0 +1,15 @@
+:root {
+  --white-color: #fff;
+  --clock-big-border-color: #34626c;
+  --background-color: #262626;
+  --bigger-pointer-height: 8px;
+  --smaller-pointer-height: 3px;
+  --bigger-pointer-width: 8px;
```

Comment:
You can use SASS variables. They don't require root and look cleaner
https://sass-lang.com/documentation/variables